### ISSUE TITLE
Newlines in XML doc <code> elements are not displayed in the IDE, creating an unreadable jumble

### ISSUE THREAD

**Issue by jnm2** [dotnet/roslyn#54414]
**Version Used**: 16.10.2, 17.0.0-p1.1

The workaround is to add `<br/>` to every line (not maintainable) and there *is* no workaround that allows there to be indentation at the start of the new lines. Maybe this is something https://github.com/dotnet/roslyn/issues/54417 can help with, but if the non-CDATA case doesn't preserve newlines and whitespace by default, then the IDE should warn and prompt to escape the contents of the `<code>` element as CDATA.

![image](https://user-images.githubusercontent.com/8040367/123515484-9254bd00-d665-11eb-8edf-a6a89eaa3bf5.png)

```cs
/// <summary>
/// <code>
/// public sealed class SomeClass
/// {
///     public void Foo(string name)
///     {
///         Console.WriteLine($"Hello, {name}!");
///     }
///
///     public bool Confirm()
///     {
///         while (true)
///         {
///             var key = Console.ReadKey(intercept: true);
///             if (key.Key == ConsoleKey.Y) return true;
///             if (key.Key == ConsoleKey.N) return false;
///             Console.Beep();
///         }
///     }
/// }
/// </code>
/// </summary>
class C { }
```

**Comment by Youssef1313** [comment]
This was fixed in https://github.com/dotnet/roslyn/pull/53140.

**Comment by jnm2** [comment]
Duplicate of #37503.
